ingestion-edge/pytest.ini

```diff
@@ -8,3 +8,8 @@ filterwarnings =
     ignore::PendingDeprecationWarning:google.cloud.pubsub_v1.publisher.client:89
     ignore::PendingDeprecationWarning:google.cloud.pubsub_v1.subscriber.client:83
 docstyle_match = (?!test_)(?!conftest).*\.py$
+env =
+    D:HOST=localhost
+    D:PORT=0
+    D:ROUTE_TABLE=[["/submit/<suffix:path>","test",["POST"]]]
+    D:QUEUE_PATH=:memory:
```

Comment:
This `D:` syntax for defaults feels a bit arcane. Can you add a comment linking to https://github.com/MobileDynasty/pytest-env#usage ?